roles/reset/tasks/main.yml

```diff
@@ -82,3 +82,8 @@
     path: /etc/rc.local
     state: absent
   when: proxmox_lxc_configure and rcfile.stat.exists and ((rcslurp.content | b64decode).splitlines() | length) <= 1
+
+- name: remove extra worker node packages
```

Comment:
This should not affect non-worker hosts, but it does.

Reply:
That's a good point, assuming we keep this limited to worker nodes, should there be a new reset role for worker nodes that has this in it? Or something else?